src/test/scala/lms/utils.scala

```diff
@@ -10,7 +10,7 @@ trait LibSuite extends FunSuite {
 }
 
 trait TutorialFunSuite extends LibSuite {
-  val overwriteCheckFiles = false // should be false; temporary set to true only to simplify development
+  val overwriteCheckFiles = true // should be false; temporary set to true only to simplify development
```

Comment:
I think we should keep it as `false`?

Reply:
I'll change it back to `true` before merge :)